### PR TITLE
refactor main.handlebars, refactor seeds api userRoutes

### PR DESCRIPTION
**Pull request by Tristenh**

### CODE REVIEW

File: views/layouts/main.handlebars [hunk 1]
```diff
@@ -1,78 +1,97 @@
...
-{{{ body }}}   
-  {{#if logged_in}}
-  <script src="/js/logout.js"></script>
-  {{/if}}    
-</main>
-  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-HwwvtgBNo3bZJJLYd8oVXjrBZt8cqVSpeBNS5n7C8IVInixGAoxmnlMuBnhbgrkm" crossorigin="anonymous"></script>
+        </div>
+        <div class='mx-auto my-2 order-0 order-md-1 position-relative'>
+          <ul class='navbar-nav'>
+            <li class='nav-item'><a href='#' class='nav-link'>Profile</a></li>
```
Comment (sakshiarora04) [inline views/layouts/main.handlebars]: delete this profile

File: views/layouts/main.handlebars [hunk 2]
```diff
@@ -1,78 +1,97 @@
...
+            <li class='nav-item'><a href='#' class='nav-link'>Contact</a></li>
+            <li class='nav-item'>{{#if logged_in}}
+                <a href='/profile' class='nav-link'>Profile</a>
+                <a href="*" class='nav-link' id='logout'>logout</a>
+              {{else}}
+                <a href='/login' class='nav-link'>login</a>
```
Comment (sakshiarora04) [inline views/layouts/main.handlebars]: login/signup 
 <a href='/login' class='nav-link'>login/signup</a>

File: views/layouts/main.handlebars [hunk 3]
```diff
@@ -1,78 +1,97 @@
...
+            type='search'
+            placeholder='Search'
+            aria-label='Search'
+          />
+          <button class='btn my-2 my-sm-0 search-btn' type='submit'><i
+              class='fa fa-search'
+            ></i></button>
+        </form>
+      </div>
+    </header>
```
Comment (sakshiarora04) [inline views/layouts/main.handlebars:85]: add div with class grid-container to apply on all pages

Comment (Tristenh) [reply views/layouts/main.handlebars:85]: applied in later push

File: views/layouts/main.handlebars (hunk 2 above)
Comment (Tristenh) [reply views/layouts/main.handlebars]: done

**Review by sakshiarora04 — APPROVED** [review]

**Pull request merged**